src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from "@apollo/client";
+import { usePreferencesAnalytics } from "analytics";
+import Icon from "components/Icon";
+import { StyledLink } from "components/styles";
+import TaskStatusBadge from "components/TaskStatusBadge";
+import { getEvergreenTaskURL } from "constants/externalURLTemplates";
+import { GetTaskQuery, GetTaskQueryVariables } from "gql/generated/types";
+import { GET_TASK } from "gql/queries";
+
+interface Props {
+  taskID: string;
+  execution: number;
+}
+
+export const EvergreenTaskSubHeader: React.FC<Props> = ({
+  taskID,
+  execution,
+}) => {
+  const { sendEvent } = usePreferencesAnalytics();
+  const { data, loading } = useQuery<GetTaskQuery, GetTaskQueryVariables>(
+    GET_TASK,
+    {
+      variables: { taskId: taskID as string, execution: Number(execution) },
```

Comment:
do we need to cast the `taskID` and `execution` if they already come in as a `string` and `number` in the props?